generators/csharp/sdk/versions.yml

```diff
@@ -1,4 +1,12 @@
 # yaml-language-server: $schema=../../../fern-versions-yml.schema.json
+- version: 2.4.1
+  changelogEntry:
+    - type: fix
+      summary: |
+        Fix nullability issues/seed failures
```

Comment:
I wouldn't mention "seed" since the changelog is for our users to read and understand. Let's focus on how it improves the SDK for users.

Reply:
fixed